README.md

```diff
@@ -101,6 +101,11 @@ spec:
 
 ## Changelog
 
+### v0.3.0
+
+#### Added
+* ability to send data before disconnect
```

Comment:
You won't be able to send the message before each disconnect. Only when the connection is closed from our side
```suggestion
* ability to send data before the WebSocket client is stopped
```